compiler/ExtractBuiltin.ml

```diff
@@ -290,6 +290,9 @@ let builtin_funs () : (pattern * bool list option * builtin_fun_info) list =
     mk_fun "core::slice::{[@T]}::len"
       (Some (backend_choice "slice::len" "Slice::len"))
       None;
+    mk_fun "core::option::{core::option::Option<@T>}::is_none"
+      (Some (backend_choice "option::is_none" "Option::isNone"))
+      (Some [ false ]);
```

Comment:
FUTUREWORK: This is a hack to avoid having an alias function to take the type explicitly, on the long run, the type should be implicit and we won't have to use the filter field as a hack.